translations/components/es-ar.yaml

```
@@ -99,6 +108,7 @@ modalAyuda:
 
 passwordConfirmation:
   password: "Confirmar contraseña"
+  matchError: "Las contraseñas no coinciden"
 
 sessionButton:
   login: "Ingresar"
```

```suggestion
  login: "Iniciar sesión"
```

Sólo porque se cambió arriba. Pero lo podemos dejar así.